src/schemas/SettingsSchema.js

```diff
@@ -3,14 +3,16 @@ const graphql = require('graphql');
 module.exports = graphql.buildSchema(`
   type Query {
     sites(siteId: String): SiteCollection
-    twitterAccounts(siteId: String!): TwitterAccountCollection
+    twitterAccounts(siteId: String!, pageState: String, pageSize: Int): TwitterAccountConnection
```

Comment:
Oh, this change in return type feels non-backwards, compatible to me. Why not add optional paging info to the `TwitterAccountCollection` type?